src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,64 @@
+- @pagetitle = 'Show Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        %p.font-weight-bold Your new OBS Personal Access Token
+
+        .form-row
+          .col-sm
+            %label.font-weight-bold
+              Id:
+            %span
+              = @token.id
+
+        - if session[:show_token]
+          - session[:show_token] = nil # Ensure we're only showing the token once
+          .form-row
+            .col-12.col-md-10.col-lg-9
+              %fieldset.form-group
+                %label.col-form-label.font-weight-bold
+                  Secret:
+                .input-group
+                  = render CopyToClipboardInputComponent.new(token_string: @token.string)
+              %hr
+
+        .form-row
+          .col
+            .form-group
+              %label.col-form-label.mr-2.font-weight-bold
+                Type:
+              %span
+                = @token.type
+
+        .form-row
+          .col-sm
+            .form-group.ui-front
+              %label.font-weight-bold
+                Name:
+              %span
+                = @token.name
+
+        - if @token.package
+          .form-row
+            .col-sm
+              .form-group.ui-front
+                %label.font-weight-bold
+                  Project name:
+                %span
+                  = @token.package.project.name
+
+          .form-row
+            .col-sm
+              .form-group.ui-front
+                %label.font-weight-bold
+                  Package name:
+                %span
+                  = @token.package.name
+
+        .actions
+          = render partial: 'actions'
```

Comment:
I don't see a need for this partial, since it only contains a single link and it's never reused. Let's put the link directly here instead. It could also be a simple link, not a button. It would look nicer by not being so "blocky".

Reply:
This partial has been used in other views as well. I've updated the button style but haven't removed the partial itself.

Reply:
You're right, it's used in this view and `src/api/app/views/webui/users/tokens/_update.js.erb`.